pkg/annotations/ingress_annotations.go

```diff
@@ -167,6 +170,11 @@ func UsePrivateIP(ing *v1beta1.Ingress) (bool, error) {
 	return parseBool(ing, UsePrivateIPKey)
 }
 
+// OverrideFrontendPort determines whether to use a custom Frontend port
```

Comment:
nit: OverrideFrontendPort provides a custom frontend port to be used in listener